doc/news/changes/minor/20190603PeterMunch

```diff
@@ -0,0 +1,6 @@
+New: The function Utilities::MPI::compute_index_owner() provides the owner of
+the ghost degrees of freedom relative to a set of locally owned ones. This
+function is used Utilities::MPI::Partitioner to avoid global all-to-all
+communication, speeding up computations on many MPI ranks.
```

Comment:
Break the second contribution up into a separate changelog (e.g. named "20190603PeterMunch-1") because it deserves to be mentioned separately, e.g. as "Improved: The function Utilities::MPI::compute_index_owner() is used Utilities::MPI::Partitioner to avoid global all-to-all communication, speeding up computations on many MPI ranks."